src/main/kotlin/github/GitHubImpl.kt

```diff
@@ -136,11 +136,40 @@ object GitHubImpl {
         files: Map<String, String>,
         packageIdentifier: String,
         packageVersion: String,
-        updateState: VersionUpdateState
+        updateState: VersionUpdateState,
+        terminal: Terminal
     ): GHPullRequest {
+        if (
+            data.PreviousManifestData.installerManifest?.equals(data.InstallerManifestData) == true &&
+            data.PreviousManifestData.defaultLocaleManifest?.equals(data.DefaultLocaleManifestData) == true &&
+            data.PreviousManifestData.versionManifest?.equals(data.VersionManifestData) == true
```

Comment:
We can just use `PreviousManifestData` and import the data part. IntelliJ IDEA should be able to do this for you if you remove the `data.`